app/code/Magento/Catalog/Model/ResourceModel/Eav/Attribute.php

```diff
@@ -383,7 +383,7 @@ public function getApplyTo()
     /**
      * Retrieve source model
      *
-     * @return \Magento\Eav\Model\Entity\Attribute\Source\AbstractSource
+     * @return mixed|string|null
```

Comment:
Why did you use here mixed instead of real type?

Reply:
Please use the following solution:
```suggestion
     * @return \Magento\Eav\Model\Entity\Attribute\Source\AbstractSource|string|null
```